### PR TITLE
Gives the user a little more feedback if the web request fails.

### PR DESCRIPTION
**Pull request by meeces2911**
eg. 401 Unauthorised because our PAT has expired!

### CODE REVIEW

File: app/requests.js [hunk 1]
```diff
@@ -15,12 +16,16 @@ module.exports = {
...
+          rejectUnauthorized: false,    // Don't validate SSL certs
           headers: opts.headers || {},
           json: true
         },
         function (error, response, body) {
-          callback(error, body);
+          if (response.statusCode === 200) {
+            callback(error, body);
+          } else {
+            callback('HTTP Reponse: '+response.statusCode+' '+http.STATUS_CODES[response.statusCode]);
```
Comment (marcells) [inline app/requests.js]: We have to be careful here. Request usually returns an error object if understand [the code](https://github.com/request/request/blob/a6741d415aba31cd01e9c4544c96f84ea6ed11e3/request.js#L209) correctly.

Please ensure, that it works in exactly the same way and _only append_ the informational error message. Then I'm fine with the change.

**Comment by meeces2911** [comment]
@marcells Does this work ? I've changed the error string into an object, and appended it if an error exists already.

**Comment by marcells** [comment]
Yea, the code looks good, but I need some time to verify it.

**Comment by marcells** [comment]
Thank you. I'll merge it.

**Pull request merged**